plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -206,14 +276,16 @@ export default class ChannelHeaderPlug extends React.PureComponent {
 
     render() {
         const components = this.props.components || [];
+        const appBindings = this.props.bindings || [];
 
-        if (components.length === 0) {
+        if (components.length === 0 && appBindings.length === 0) {
             return null;
-        } else if (components.length <= 5) {
-            return components.map(this.createButton);
+        } else if (components.length + appBindings.length <= 5) {
+            const componentButtons = components.filter((plug) => plug.icon && plug.action).map(this.createComponentButton);
```

Comment:
The current behaviour if no icon would be showing a blank button. If no action is provided, will be a non-responsive button. Because of that, I feel it will not be an actual regression.